docs/modules/lists/pages/ordered.adoc

```diff
@@ -43,7 +43,9 @@ If your ordered list includes figures or independent chunks of text, as is often
 This may happen, for example, after figures or multiparagraph steps. 
 In these cases, don't use the `start` attribute to continue the numbering.
 You don't want to have to manually update `start` numbers—or risk having the `start` numbers become wrong—every time you add or remove a step.
-Instead, use a list continuation—a `{plus}` symbol on its own line—between the numbered list items and all other content elements in your list.
+Instead, use a list continuation—a `{plus}` symbol on its own line—between the numbered list items and all other content elements in your list, as shown here:
```

Comment:
This sentence needs to be reworded slightly since the `+` isn't needed between the numbered list items, only between block elements (multiple paragraphs, images, tables, source blocks, etc.) that are separated by blank lines within a list item.

Reply:
Done. Thanks!

Reply:
The edits looks great! I'm going to merge this in now. Thank you for the updates and taking the time to do the image, Marcia.